### PR TITLE
fix: imports order in .brs files stopped working

### PR DESCRIPTION
**Pull request by bchelkowski**
BREAKING CHANGE: Changed order of imports

## What did you implement:

Closes [#7](https://github.com/getndazn/kopytko-eslint-plugin/issues/7)

Fixes dependency order checks after adding the possibility to use external packages in the kopytko-packager.

## How can we verify it:

Checked locally.

## Todos:

- [x] Write documentation (if required)
- [x] Enable "Allow edits from maintainers" for this PR
- [x] Update the messages below

***Is this ready for review?:*** YES
***Is it a breaking change?:*** YES

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,23 +106,91 @@ and [Kopytko Unit Testing Framework](https://github.com/getndazn/kopytko-unit-te
...
+```
+
+Example of **correct** code:
+```brightscript
+' @import /components/some-function.brs from another-package-name
+' @import /components/a-function.brs from package-name
+' @import /components/main-function.brs
+' @mock /components/some-mocked-function.brs from another-package-name
+' @mock /components/another-mocked-function.brs from package-name
+' @mock /components/mocked-function.brs
```
Comment (tomek-r) [inline README.md:173]: Wondering if it wouldn't be better to have external imports first, then the rest. What do you think?

Comment (bchelkowski) [reply README.md:173]: yeah, maybe it would be even more readable?

Comment (tomek-r) [reply README.md:173]: Actually, I am not sure. Probably this is a matter of subjective preference.

Comment (bchelkowski) [reply README.md]: or maybe add two rules, and anyone can use what suits them better?

Comment (tomek-r) [reply README.md]: If we change it will we have to fix all the occurances?

Comment (bchelkowski) [reply README.md]: depends on the implementation, I won't add it here, so we need to decide which way we want to go

**Review by tomek-r — APPROVED** [review]

**Pull request merged**